src/Data/Formatter/DateTime.purs

```diff
@@ -210,16 +225,26 @@ padQuadrupleDigit i
   | i < 1000 = "0" <> (show i)
   | otherwise = show i
 
+-- | Format a DateTime according to the format defined in the given `Formatter`
 format ∷ Formatter → DT.DateTime → String
 format f d = foldMap (formatCommand d) f
 
+-- | Format a DateTime according to the format defined in the given format string.
+-- | If the format string is empty, will return a `Left` value. Note that any 
+-- | unrecognized character is treated as a placeholder, so while "yyyy-MM-dd" might
+-- | not produce the format you want (since "yyyy" and "dd" aren't recognized formats),
```

Comment:
This statement would become inaccurate if #37 were implemented and merged.

Reply:
I'll go ahead and change that too while we're sanding off rough edges. The comment still holds true for `yyyy` though.

Reply:
I'll update this comment with the assumption that "d" will be an accepted format then.